modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/RightSidebar/RightSidebarObjectRelationshipDetails.tsx

```diff
@@ -197,12 +201,9 @@ export function RightSidebarObjectRelationshipDetails({
 						}
 					);
 
-					if (
-						(element as Edge<ObjectRelationshipEdgeData>).data
-							?.objectRelationshipId === objectRelationship?.id
-					) {
+					if (objectRelationshipId === objectRelationship?.id) {
```

Comment:
If both `objectRelationshipId` and `objectRelationship?.id` are undefined, then this will evaluate to true and the conditioned code will run. Is this a problem? How likely it is for both this values come as undefined?

Reply:
I believe this could cause problems, so I've added a commit to prevent this. Thanks @cgmonte !